csrc/quantization/cutlass_w8a8/scaled_mm_c3x.cuh

```diff
@@ -0,0 +1,468 @@
+#pragma once
```

Comment:
Should we try to separate the epilogues out as well into a separate header file?

Reply:
https://github.com/vllm-project/vllm/pull/9855 did that. I have rebased this PR.